datareduction/datareductionoperator.cpp

```diff
@@ -740,38 +701,34 @@ namespace DRO {
    
    
    // YK minimum value of the distribution function
-   MinDistributionFunction::MinDistributionFunction(cuint _popID): DataReductionOperator(),popID(_popID) {
-      popName=getObjectWrapper().particleSpecies[popID].name;
+   MinDistributionFunction::MinDistributionFunction(cuint popID): DataReductionOperator() {
+     _popID = popID;
+     _name=getObjectWrapper().particleSpecies[_popID].name;
    }
    MinDistributionFunction::~MinDistributionFunction() { }
    
-   std::string MinDistributionFunction::getName() const {return popName + "/MinimumDistributionFunctionValue";}
+   std::string MinDistributionFunction::getName() const {return _name + "/MinimumDistributionFunctionValue";}
    
    bool MinDistributionFunction::getDataVectorInfo(std::string& dataType,unsigned int& dataSize,unsigned int& vectorSize) const {
       dataType = "float";
       dataSize =  sizeof(Real);
       vectorSize = 1;
       return true;
-   }
-   
+   }   
    
    bool MinDistributionFunction::reduceDiagnostic(const SpatialCell* cell,Real* buffer) {
-      const Real HALF = 0.5;
       minF =  std::numeric_limits<Real>::max();
 
       #pragma omp parallel 
       {
          Real threadMin = std::numeric_limits<Real>::max();
          
-         const Realf* block_data = cell->get_data(popID);
+         const Realf* block_data = cell->get_data(_popID);
 
          #pragma omp for
-         for (vmesh::LocalID n=0; n<cell->get_number_of_velocity_blocks(popID); ++n) {
-            for (uint k = 0; k < WID; ++k)
-               for (uint j = 0; j < WID; ++j)
-                  for (uint i = 0; i < WID; ++i) {
-                     const int celli=k*WID*WID+j*WID+i;
-                     threadMin = min((Real)(block_data[n * SIZE_VELBLOCK+celli]), threadMin);
+         for (vmesh::LocalID n=0; n<cell->get_number_of_velocity_blocks(_popID); ++n) {
+	    for (uint k = 0; k < WID; ++k) for (uint j = 0; j < WID; ++j) for (uint i = 0; i < WID; ++i) {
```

Comment:
This (and similar loops, I only comment here now) actually goes against our code style policy of always having indentations and curly brackets for if and loop structures to avoid confusion w.r.t. implicit/assumed blocks even for one-liners. But I don't mind in this case.